homeassistant/components/geniushub/config_flow.py

```diff
@@ -0,0 +1,179 @@
+"""Config flow for Geniushub logger integration."""
+
+from __future__ import annotations
+
+from http import HTTPStatus
+import logging
+import socket
+from typing import Any
+
+import aiohttp
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+
+from . import V1_API_SCHEMA, V3_API_SCHEMA, validate_input
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GeniusHubConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Solis logger."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """User config step for determine v1 or v3."""
+        _LOGGER.debug("ConfigFlow.async_step_user")
+        return self.async_show_menu(
+            step_id="user",
+            menu_options=["v3_api", "v1_api"],
```

Comment:
Oh and I would still be a big fan of naming these options to what they are, thus local and cloud. Otherwise developers new to the code always have to check strings.json first to see the full picture

Reply:
Done.